packages/components/src/molecules/ProductCard/ProductCardContent.tsx

```diff
@@ -92,22 +95,37 @@ const ProductCardContent = forwardRef<HTMLElement, ProductCardContentProps>(
           </h3>
           {!outOfStock && (
             <div data-fs-product-card-prices>
-              <Price
-                value={price?.listPrice ? price.listPrice : 0}
-                formatter={price?.formatter}
-                testId="list-price"
-                data-value={price?.listPrice}
-                variant="listing"
-                SRText="Original price:"
-              />
-              <Price
-                value={price?.value ? price.value : 0}
-                formatter={price?.formatter}
-                testId="price"
-                data-value={price?.value}
-                variant="spot"
-                SRText="Sale Price:"
-              />
+              {sellingPrice !== listPrice
+                ? (
+                  <>
+                    <Price
+                      value={listPrice}
+                      formatter={price?.formatter}
+                      testId="list-price"
+                      data-value={listPrice}
```

Comment:
You are changing behavior here. Previously, `listPrice` (and sellingPrice) could be `undefined`. Now, in those cases, it will be 0, which may lead to unforeseen consequences.

Reply:
You're talking about the `data-value` attribute, right? I've searched the project and don't see it being used in places other than tests. Keep in mind that the `value` attribute was being set to 0 via the ternary operator here: `price?.listPrice ? price.listPrice : 0`

Reply:
Yes, makes sense! It also makes sense to be equal to the actual value, so no problem.